akka-actor/src/main/scala-2.12/scala/annotation/nowarn.scala

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+// TODO: remove this when upgrading to Scala 2.12.13
+// https://github.com/scala/scala/pull/9248
+package scala.annotation
```

Comment:
will this work with OSGi? I think it doesn't allow source in other packages than the root package of the module.
Eventually we will remove the  OSGi support https://github.com/akka/akka/issues/28304 but  we are not there yet.

Reply:
How can we test?
I was working under the assumption that, if the tests are passing, I'm not breaking anything.

That said, I'm not sure how can OSGi detect this, at best of my understanding:
 - Akka depends on Scala and scala library should be on the classpath
 - I'm not aware of a way to distinguish from which `jar` a specific class is coming from given a classpath
 - `@nowarn` is an annotation and should not spill to runtime
 
Am I making any wrong assumptions @patriknw ? I'm certainly not an OSGi expert ...